src/include/os.h

```diff
@@ -6,18 +6,16 @@
  * See the file LICENSE for redistribution information.
  */
 
-#define	WT_SYSCALL_RETRY(call, ret) do {				\
-	int __retry;							\
-	for (__retry = 0; __retry < 10; ++__retry) {			\
-		if ((call) == 0) {					\
-			(ret) = 0;					\
-			break;						\
-		}							\
-		switch ((ret) = __wt_errno()) {				\
-		case 0:							\
-			/* The call failed but didn't set errno. */	\
-			(ret) = WT_ERROR;				\
-			break;						\
+/*
+ * System call macro to retry system/library calls on "expected" errors.
+ * Three versions: one of which looks for a -1 return value on error and
```

Comment:
I'd prefer to see this comment formatted as a numbered or bulleted list of the three versions rather than a dense paragraph.